### ISSUE TITLE
RFU not working

### ISSUE THREAD

**Issue by c0mbat4shooterlol** [axstin/rbxfpsunlocker#1892]
uninstalled roblox, console is saying everything is fine, re-installed roblox, uninstalled RFU, reinstalled RFU roblox is still capped at 60fps and wont go above, not a vsync issue as nvidia control panel says its capped at 400fps and vsync is off and im on the latest version of RFU (4.3.0)

**Comment by ShadowmanWasTaken** [comment]
Try reinstalling your NVIDIA graphics card driver and make sure it is the latest version.

**Comment by agentisnotreal** [comment]
Was experiencing this earlier today, but updating/reinstalling my Nvidia driver seemed to do the trick.

**Comment by c0mbat4shooterlol** [comment]
> 
> 
> Was experiencing this earlier today, but updating/reinstalling my Nvidia driver seemed to do the trick.



> 
> 
> Try reinstalling your NVIDIA graphics card driver and make sure it is the latest version.

I did install my newest nvidia GPU drivers, FPS still capped at 60

**Comment by agentisnotreal** [comment]
Does the RFU system tray show that it is attached to a process?
![image](https://user-images.githubusercontent.com/55689844/101243267-dc407a80-36f6-11eb-9743-660d57bd3c4d.png)

**Comment by c0mbat4shooterlol** [comment]
> 
> 
> Does the RFU system tray show that it is attached to a process?
> ![image](https://user-images.githubusercontent.com/55689844/101243267-dc407a80-36f6-11eb-9743-660d57bd3c4d.png)

Yeah it shows an attached process and I've checked it launches the right version of ROBLOX
